test_runner/src/main/kotlin/ftl/shard/Shard.kt

```diff
@@ -35,6 +36,16 @@ class com.foo.ClassName#testMethodToSkip
 
 */
 
+fun AndroidArgs.createShardsByTestForShards() = this.testTargetsForShard.map { list ->
```

Comment:
It's much better to expose return type instead of `this` especially when function is public
```suggestion
fun AndroidArgs.createShardsByTestForShards(): List<Chunk> = testTargetsForShard.map { list ->

```

Reply:
Already resolved.